### ISSUE TITLE
Gradle / Maven Coordinates

### ISSUE THREAD

**Issue by JuKu** [libgdx/gdx-video#51]
With gradle i get the following build error:
```
Could not find com.badlogicgames.gdxvideo:gdx-video:0.0.1.
Searched in the following locations:
    https://repo1.maven.org/maven2/com/badlogicgames/gdxvideo/gdx-video/0.0.1/gdx-video-0.0.1.pom
    https://repo1.maven.org/maven2/com/badlogicgames/gdxvideo/gdx-video/0.0.1/gdx-video-0.0.1.jar
```

Why this JARs arent available in Maven Central anymore?

**Comment by JuKu** [comment]
@RBogie Can anyone fix this issue, please?

**Comment by RBogie** [comment]
Personally I can't add jars to maven central. This would probably be something for @Tom-Ski

**Comment by JuKu** [comment]
@RBogie If this is the only problem, i can publish them to maven central.
But the native files are currently missing, that's the real problem.

**Comment by JuKu** [comment]
And can you update to newest libGDX version, pls?

**Comment by SimonIT** [comment]
Fixed with ce4049660833c387728d7dcd5bf79418f2c585a9
